### ISSUE TITLE
When scheduling a recurring meeting in the future, ensure name of first retro uses first date of meeting

### ISSUE THREAD

**Issue by charlesmathias** [ParabolInc/parabol#10111]
[Steps to replicate](https://parabol.slack.com/archives/C4JAUUZ9P/p1723625575001099?thread_ts=1723625526.311249&cid=C4JAUUZ9P) 🔒

Impacts both standups and retrospectives.

**Comment by jordanh** [comment]
See also: #10113

**Comment by Dschoordsch** [comment]
@charlesmathias If you schedule a recurring meeting for Aug 16, do you still want it to be opened straight away?

The issue is this would be inconsistent, because right now we put the start time in the meeting name. If you create a series only repeating on Thursdays on Aug 13, then you get a meeting which starts straight away but will be closed on Aug 16 and a new one is started. Alternative would be a gap or an exception which would be a meeting lasting longer than 1 week.

**Comment by charlesmathias** [comment]
I understand. To get to this view, users need to press "Schedule". I believe users would expect it to open when they've scheduled the first meeting, so Aug 16 in this case.

There are two things happening:
1. Setting the date, time and recurrence of a sync meeting
2. The opening and closing of Parabol meetings

It isn't clear right now how these interact with one another.

I schedule a future retro for Aug 16, 3 days in advance, and set up recurrence for every two weeks. It isn't clear to me that a retro is going to open straight away and will automatically close on Aug 16.

Perhaps a fix would be to update the communication around what happens when a user schedules a recurring meeting: eg the first meeting is created straight away, and then recurs from the first scheduled date.

**Comment by Dschoordsch** [comment]
Yes, it's confusing because both features are independent, but interact with each other in real life.

I think a gap would be valid and easy enough to implement. So in start meeting we would just setup the meeting series and scheduling, but not starting the current instance.

**Comment by jordanh** [comment]
This has been fun to think about. I have a proposal. Could we resolve https://github.com/ParabolInc/parabol/issues/9748 https://github.com/ParabolInc/parabol/issues/10111 https://github.com/ParabolInc/parabol/issues/10113 by doing the following:

- When creating a new meeting series, offer a configuration option, "Start first meeting now?"
- When a meeting is created from a recurrence, always name it [Team] Retro ending [Date]
